com.unity.render-pipelines.high-definition/Documentation~/shared/lens-flare/lens-flare-component.md

```diff
@@ -43,4 +43,6 @@ To create a lens flare in a scene:
 | Occlusion Radius | Defines how far from the light source Unity occludes the lens flare. This value is in world space. |
 | Sample Count | The number of random samples the CPU uses to generate the **Occlusion Radius.** |
 | Occlusion Offset | Offsets the plane that the occlusion operates on. A higher value moves this plane closer to Camera. This value is in world space. <br/>For example, if a lens flare is inside the light bulb, you can use this to sample occlusion outside the light bulb. |
+| Volumetric Cloud Occlusion | When enabled, HDRP uses the volumetric cloud texture for the occlusion. Because the occlusion is computed in screen space, the lens flare is always considered behind the volumetric clouds.<br/>This option is only visible if Volumetric Could is enabled on the HDRP Asset. |
```

Comment:
I suggest changing the description: 

When enabled, HDRP uses the volumetric cloud texture to occlude the lens flare. HDRP always considers the lens flare to be behind the volumetric clouds because it calculates occlusion in screen space.